### PR TITLE
Implement placeholder testing to be run in CI

### PR DESCRIPTION
**Pull request by M3kH**
Contributes to the the planned issue of #8 

Delivers just placeholder because I'm still not familiar with the codebase.
And was the smallest step.

---

Create placeholder test for:
- DgcLibAutoConfigurationTest
- SignedCertificateMessageBuilderTest
- CertficateUtilsTest

Add test runner on CI for pull request.

### CODE REVIEW

File: .github/workflows/ci-pr.yml [hunk 1]
```diff
@@ -16,4 +16,6 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
       - name: Build package
-        run: mvn install
\ No newline at end of file
+        run: mvn install
+      - name: Test package
```
Comment (M3kH) [inline .github/workflows/ci-pr.yml:20]: I believe this is not necessary because mvn runs them at install

Comment (f11h) [reply .github/workflows/ci-pr.yml:20]: yes, thats right

**Comment by f11h** [comment]
@M3kH currently I'm also working on the Unit tests. I would like to merge your PR into my local dev branch. Then your contribution is not lost.

Is it okay for you? Then I would close this PR.

**Comment by M3kH** [comment]
> Is it okay for you? Then I would close this PR.

Yes sure go ahead! :-)

**Pull request closed without merging**